projects/ion/src/lib/alert/alert.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { CommonModule } from '@angular/common';
+import { IonIconComponent } from './../icon/icon.component';
+import { render, screen, fireEvent } from '@testing-library/angular';
+import { AlertComponent, IonAlertProps } from './alert.component';
+import { StatusType } from '../core/types/status';
+
+const defaulValue: IonAlertProps = {
+  message: 'Mensagem padrão',
+};
+
+const alertIDs = {
+  alert: 'ion-alert',
+  iconStatus: 'status-icon',
+  iconClose: 'close-icon',
+};
+
+const closableAlert = 'closable-true';
+
+const alertTypes = ['success', 'warning', 'info', 'negative'];
+
+const sut = async (customProps?: IonAlertProps) => {
+  await render(AlertComponent, {
+    componentProperties: customProps || { ...defaulValue },
+    declarations: [IonIconComponent],
+    imports: [CommonModule],
+  });
+  return screen.findByTestId(alertIDs.alert);
+};
+
+describe('AlertComponent', () => {
+  it('Should render alert', async () => {
+    expect(await sut()).toHaveClass(alertIDs.alert);
+  });
+
+  it('Alert should have default message', async () => {
```

Comment:
```suggestion
  it('Alert should have a message', async () => {
```

Because the component dont have a default message, message is required.